tic_tac_toe/src/board_test.py

```diff
@@ -0,0 +1,162 @@
+from board import Board
+
+
+def validate_move_update():
+    board = Board(3)
+    board.values[0] = "X"
+    return board
+
+
+def test_row_1():
```

Comment:
Name these tests something sensible, not numerically.  I have no idea what `test_row_1` is doing from reading the name.  I should know precisely what the test case is from reading the function name.